lib/common/presentation/views/faq/faq_screen.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
+
+/// The page [FaqScreen]
+class FaqScreen extends StatelessWidget {
+  /// Constructs an instance of [FaqScreen]
+  const FaqScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) => Padding(
+        padding: const EdgeInsets.all(40),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Row(
+              children: [
+                Image.asset(
+                  'assets/images/speechbubbleimage.png',
```

Comment:
Use the asset image folder constant to access your images.

`Image.asset('$assetImagePath${your-image-name}')`